package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "scripts": {
     "lint": "npm run lint:apex ; npm run lint:doc",
-    "lint:apex": "sfdx scanner:run -v -t 'force-app/classes/src/*.cls' -f table -c 'apex-mockery'",
+    "lint:apex": "sfdx scanner:run -v -t 'force-app/classes/**/*.cls' -f table -c 'apex-mockery'",
```

Comment:
**remarks:** I think it is still not working...

we don't have force-app/classes folder ...
we should let this [PR](https://github.com/salesforce/apex-mockery/pull/23) fix it